src/libpgexporter/configuration.c

```diff
@@ -163,6 +163,17 @@ pgexporter_read_configuration(void* shm, char* filename)
                memcpy(&section, line + 1, max);
                if (strcmp(section, "pgexporter"))
                {
+
+                  for (int j = 0; j < idx_server - 1; j++)
```

Comment:
Why -1 ?

Reply:
So, in `i`th iteration, the `i-1`th server (`srv`) is getting saved ([here](https://github.com/pgexporter/pgexporter/blob/21de355ec68e6da5e8ab6a86ef7ba0d4872a084a/src/libpgexporter/configuration.c#L187) and [here](https://github.com/pgexporter/pgexporter/blob/21de355ec68e6da5e8ab6a86ef7ba0d4872a084a/src/libpgexporter/configuration.c#L179)). So, if I need to compare `srv.name` to check duplication of names, I need to check for `0` to `i-2` including both.